css/styles.scss

```diff
@@ -0,0 +1,257 @@
+@import url(//db.onlinewebfonts.com/c/3a47f5f06b5484abfeee9eac90348a9c?family=Gotham);
+@import "./variables/colors";
+@import "./variables/extends";
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Gotham";
+  overflow-x: hidden;
+}
+
+/*Navbar*/
+.navbar {
+  width: 98vw;
+  height: 8vh;
+  margin-left: 1rem;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  .left-sector {
+    margin: 0;
+    padding: 0;
+    .tesla-logo {
+      height: 1.8rem;
+    }
+  }
+  .middle-sector {
+    margin-left: 6rem;
+    .category-list {
+      width: 100%;
+      margin: 0;
+      padding: 0;
+      display: flex;
+      flex-direction: row;
+      li {
+        margin-right: 1rem;
+        list-style: none;
+        font-size: 14px;
+        a {
+          color: #000;
+          text-decoration: none;
+        }
+      }
+    }
+  }
+  .right-sector {
+    .shop-list {
+      width: 100%;
+      margin: 0;
+      padding: 0;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      li {
+        margin-right: 1.5rem;
+        list-style: none;
+        font-size: 14px;
+        a {
+          color: #000;
+          text-decoration: none;
+        }
+      }
+    }
+    /*Menu Button*/
+    .menu-btn {
+      display: inline-block;
+      cursor: pointer;
+    }
+
+    .bar1,
+    .bar2,
+    .bar3 {
+      width: 20px;
+      height: 2px;
+      background-color: #333;
+      margin: 3px 0;
+      transition: 0.7s;
+    }
+    .change .bar1 {
+      -webkit-transform: rotate(-45deg) translate(-9px, 6px);
+      transform: rotate(-45deg) translate(-9px, 6px);
+    }
+    .change .bar2 {
+      opacity: 0;
+    }
+    .change .bar3 {
+      -webkit-transform: rotate(45deg) translate(-8px, -8px);
+      transform: rotate(45deg) translate(-8px, -8px);
+    }
+  }
+}
+
+/*First Sector*/
+.first-sector {
+  width: 100vw;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url(../assets/First-Model-S-cropped.jpg);
+  background-size: cover;
+  @extend %sectors;
+}
+/*Second Sector*/
+.second-sector {
+  width: 100vw;
+  height: 100vh;
```

Comment:
If the `width: 100vw` and `height: 100vh` thing repeats thruought your app, it might be better to make it a mixin